src/main/java/de/retest/surili/commons/actions/ChangeValueOfAction.java

```diff
@@ -26,7 +26,7 @@
 	public String toString() {
 		return ChangeValueOfAction.class.getSimpleName() //
 				+ "(" //
-				+ "path=" + targetElement.getIdentifyingAttributes().getPath() //
+				+ targetElement.toString() //
```

Comment:
`Element#toString()` calls `IdentifyingAttributes#toString()`. If there is no `text`, this only prints the type, which can be quite ambiguous. What I like about the (X)Path approach is that one can easily find a string representation in the browser. What do you think?

Also: You now sometimes use keys / field names, sometimes not. For example:

```
ChangeValueOfAction(input, keysToSend=[foo])
```

To be consistent, I'd rather keep them:


```
ChangeValueOfAction(targetElement=input, keysToSend=[foo])
```

Here and also elsewhere. This also corresponds to what Lombok or most IDEs generate.

Reply:
Ok